index.html

```diff
@@ -17,17 +17,19 @@
   <body>
 
    <div id="everythingElse">
-          		<div class= "header"></div>	
+  		<div class= "header"></div>	
 
-        		<div class= "go-away">
+  		<div class= "go-away">
 
-        			  <img src="bkg.jpg"> 
-        				<div id= "container1">
-        				<div id= "song vomit"><img src="SONG VOMIT.png"></div>
-        				<h1>Enter word and a song with that word will appear</h1>
-         				<input id="query" type="text" placeholder="Enter word"></input>
-         				<p><button class="aButton" id="search" data-apiTerm="salad">Search</button></p>
-         		</div>		
+  			  <img src="bkg.jpg"> 
+  				<div id= "container1">
+    				<div id= "song vomit"><img src="SONG VOMIT.png"></div>
+    				<h1>Enter word and a song with that word will appear</h1>
+     				<input id="query" type="text" placeholder="Enter word"></input>
+     				<p><button class="aButton" id="search" data-apiTerm="salad">Search</button></p>
+       		</div>	
+      </div>	
+    </div>
    
```

Comment:
You didn't have your div's closed correctly and had your canvas part of the larger div that you hide once you get results. Thats why you haven't been seeing anything- you can check that in the web inspector